### DISCUSSION TITLE
Is it possible to point the edit and view urls to a different repo?

### DISCUSSION THREAD

**Discussion by MagicLike** [squidfunk/mkdocs-material#4820] [Q&A] [answered]
My current MkDocs `repo_url` points to a different repo than the repo where the docs are stored. Is it possible to point the edit and view URLs towards the docs repo instead of the project repo?

**Comment by squidfunk** [comment, marked as answer]
Yes, you could override the [`actions`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/actions.html) partial and adjust the URLs manually, specifically here:

https://github.com/squidfunk/mkdocs-material/blob/c8fb426d988e9cdf75054347e4152ac0063528e1/src/partials/actions.html#L38-L48

Read our [customization guide](https://squidfunk.github.io/mkdocs-material/customization/) to learn how to [override partials](https://squidfunk.github.io/mkdocs-material/customization/#overriding-partials).

**Reply by MagicLike** [reply]
Ok, thank you! I actually did not have to do this, I just found out, that I can put a full URL as the `edit_uri` value...

**Reply by MagicLike** [reply]
But wouldn't it be better to use `blob` instead of `raw`?

**Reply by squidfunk** [reply]
What's the benefit?

**Reply by MagicLike** [reply]
That the file can be viewed as compiled markdown as well as a raw file.

**Reply by squidfunk** [reply]
The idea behind the "view source" button is that you can view the Markdown sources directly.

**Reply by MagicLike** [reply]
Ahhh ok, I just thought of bringing in extra functionality, because the `blob` option can do both...
